code/drasil-example/Drasil/DblPendulum/Body.hs

```diff
@@ -181,6 +188,31 @@ tMods = [accelerationTM, velocityTM, newtonSL, newtonSLR]
 physSystParts :: [Sentence]
 physSystParts = map ((!.) . atStartNP) [the rod, the mass]
 
+-- References --
+citeRefs :: [Reference]
+citeRefs = map rw citations
+
+tModRefs :: [Reference]
+tModRefs = map rw tMods
+
+bodyRefs :: [Reference]
+bodyRefs = map rw concIns
+
+-- below are references used in all examples
+srsRefs :: [Reference]
+srsRefs = map rw [SRS.reference ([]::[Contents]) ([]::[Section]), 
```

Comment:
Would a possible fix be adding a new file (maybe in Drasil/DocLang) that could group all of these common references in a single list?

Reply:
The 'reference data', from which you can reconstruct a `Reference`, yes.

Reply:
If I were to construct reference labels for all the sections in Drasil.DocLang.SRS, I don't think we would have any need for the `section'` function (which means we can get rid of `toString` for now). This would mean that I would have to manually write all of the label names even though NamedIdeas for them exist. However, once `shortname` is changed, this should resolve nicely. 

For now, even though it looks like we're going backwards, I think I should manually write out all of the section labels in order to get at the reference information needed (there's already a bunch of them done too). Then, when `shortname` is changed, switch to using `NamedIdeas` for the labels (which will be much easier since I'll be organizing them now). Do you think this is a good idea?

Reply:
I tried to implement some of these changes. Let me know what you think. Also, in Data.Drasil.Concepts.Documentation at lines 260-261, I could replace the NamedChunk definition at lines 234-235 but it breaks stable and the labels don't really look nice. Hopefully this can be changed once #1142 is resolved.

Reply:
So yes, I think that's the way to go. And yes, often some refactors look like they first take a step backwards to better enable a good step forward.  I'll look at the code next.